src/test/controller/PromotionController.test.ts

```diff
@@ -377,17 +313,109 @@ describe('Unit tests for PromotionController', function () {
       });
   });
 
-  async function cacheLatLonForPromotions(promotions: Promotion[]) {
-    for (const promotion of promotions) {
-      if (promotion.lat && promotion.lon) {
-        await cachingService.cacheLatLonValues(
-          promotion.placeId,
-          promotion.lat,
-          promotion.lon
+  test('GET /promotions/:id/restaurantDetails/:placeId', async (done) => {
```

Comment:
Should this be
```suggestion
  test('GET /promotions/:id/restaurantDetails', async (done) => {
```

and same for the other test titles?

Reply:
Yes good catch, I must have missed this change when I changed the endpoint. Thanks!